x/stake/keeper/delegation.go

```diff
@@ -95,6 +95,26 @@ func (k Keeper) GetUnbondingDelegation(ctx sdk.Context,
 	return ubd, true
 }
 
+// load all unbonding delegations from a particular validator
+func (k Keeper) GetUnbondingDelegationsFromValidator(ctx sdk.Context, valAddr sdk.Address) (unbondingDelegations []types.UnbondingDelegation) {
```

Comment:
as mentioned later, maybe this should take a function to operate on each unbonding delegation?

Reply:
See above comment/question.